webpack/assets/javascripts/react_app/components/common/MessageBox.js

```diff
@@ -0,0 +1,41 @@
+// temporary component
+// will be replaced by patternfly markup when available
+import React from 'react';
+
+const messageBoxStyles = {
```

Comment:
:-1: for including style inside here, at least put it in a separate file to be consistent with the rest of the PR. I do think the style discussion should probably be delayed to a future PR.